requirements.txt

```diff
@@ -1 +1,5 @@
-pfzy==0.3.4
\ No newline at end of file
+pfzy>=0.3.4
+ovos-utils
+ovos-bus-client
+ovos-workshop~=0.0.12a7
```

Comment:
If this is for a stable release and it requires newer than `0.0.11`, please pin a max version `<=0.0.12` to prevent future updates to alpha versions

Reply:
Sorry, still needs the minimum spec if you need that alpha, so full string would be `>=0.0.12a7,<0.0.12`